rootfs/Dockerfile

```diff
@@ -31,4 +31,4 @@ WORKDIR /app
 CMD ["/app/bin/boot"]
 EXPOSE 8000
 
-ENV WORKFLOW_RELEASE 2.0.0
+ENV WORKFLOW_RELEASE 2.1.0
```

Comment:
sounds good. This was also semi-useful for determining what version of the server was installed using `deisctl`, but now users can just lookup the image tag in the kubernetes pod manifest making this moot. A meta issue to remove these would be great. :)